### PR TITLE
Feat(migration):Added migrations to create tables 

### LINKED ISSUE #10: [FEATURE]: Add migrations to create users or admins tables
## Description

Research and find a tool to write migrations. For example, [Alembic](https://alembic.sqlalchemy.org/en/latest/).

### PR DESCRIPTION
**Pull request by cornhubius**
**What**

- added migrations to create users or admins tables
- directory `migration` added to ignore(flake8)
- added README
- added in requirements `flask-migrate, flask-sqlalchemy`
- added classes describing tables(SQLAlchemy)
- added universal path to DB

closed #10

### CODE REVIEW

File: backend/migrations/README [hunk 1]
```diff
@@ -0,0 +1,14 @@
+Single-database configuration for Flask.
```
Comment (mentortechabc) [inline backend/migrations/README:1]: Could you please add an extension to the name of this file? I think that would help different tools to display it nicely

Comment (mentortechabc) [reply backend/migrations/README:1]: Also I would say we need here two sections for the ductumentation:

## How to set up a database from scratch 
Starting from `git clone ...` of the repo, how to set up a working database on a local machine.

## How to create a new migration (version of the database)
For example if we need to add a table.

Comment (mentortechabc) [reply backend/migrations/README:1]: And then mention this document in the main backend readme.

**Review by DramaTourG — APPROVED** [review]

File: backend/migrations/README.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+Single-database configuration for Flask.
```
Comment (mentortechabc) [inline backend/migrations/README.md:1]: Perfect! Thanks for updating the doc!

BTW github can render md (markdown) files and this is how it looks like:
https://github.com/meetingbook/meetingbook/blob/25b13de22a29da851831f6cc8e68c64ca7b8d1e7/backend/migrations/README.md

Comment (mentortechabc) [reply backend/migrations/README.md:1]: image

**Review by mentortechabc — APPROVED** [review]

**Pull request merged**